src/app/backend/resource/secret/secretdetail.go

```diff
@@ -51,11 +52,24 @@ func GetSecretDetail(client *client.Client, namespace, name string) (*SecretDeta
 	return getSecretDetail(rawSecret), nil
 }
 
+func hideSecretData(rawSecret *api.Secret) map[string][]byte {
+	// For opaque and basicAuth secret types the credentials like username, password and etc. should be hidden.
+	if rawSecret.Type == api.SecretTypeOpaque || rawSecret.Type == api.SecretTypeBasicAuth {
+		data := make(map[string][]byte)
+		for key, value := range rawSecret.Data {
+			data[key] = []byte(fmt.Sprintf("%d bytes", len(value)))
```

Comment:
Hmm... This is not translatable. Can you send the data and render it on the client as the `FOO bytes`? 

And can you make sure to enable the option of rendering this fully? I.e., `foo-secret: 1234 bytes (show value)`. This is because we cannot entirely hide the value of the secret. Hiding it by default is nice, but entirely, not that much. 

@maciaszczykm @digitalfishpond What do you think?